server/src/main/java/org/apache/kafka/network/SocketServerConfigs.java

```diff
@@ -77,8 +77,8 @@ public class SocketServerConfigs {
     public static final String ADVERTISED_LISTENERS_CONFIG = "advertised.listeners";
     public static final String ADVERTISED_LISTENERS_DOC = String.format("Specifies the listener addresses that the Kafka brokers will advertise to clients and other brokers." +
                     " The config is useful where the actual listener configuration <code>%s</code> does not represent the addresses that clients should" +
-                    " use to connect, such as in cloud environments. In environments using ZooKeeper, these addresses are published to ZooKeeper." +
-                    " In Kraft mode, the address would be published to and managed by kraft controller, the brokers would pull these data from controller as needed." +
+                    " use to connect, such as in cloud environments." +
+                    " The address would be published to and managed by the KRaft controller, the brokers pull these data from the controller as needed." +
```

Comment:
Actually I think we should use present tense. Something like `The addresses are published ...`

Reply:
Oh sorry, this is my bad